charmcraft.yaml

```diff
@@ -6,14 +6,8 @@
 type: charm
 bases:
   - build-on:
-      - name: "ubuntu"
-        channel: "20.04"
+      - name: ubuntu
+        channel: "22.04"
     run-on:
-      - name: "ubuntu"
-        channel: "20.04"
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
+      - name: ubuntu
+        channel: "22.04"
```

Comment:
So we don't support `20.04` anymore?

Reply:
I had some issues with CI when defining multiple build-ons and run-ons, removed the additional `build-on` target and defined two `run-on` instead and it seems to be working fine.